src/util/warcwriter.ts

```diff
@@ -47,6 +55,8 @@ export class WARCWriter implements IndexerOffsetLength {
   }) {
     this.archivesDir = archivesDir;
     this.tempCdxDir = tempCdxDir;
+    // for now, disabling CDX
+    this.tempCdxDir = undefined;
```

Comment:
Maybe should undo this/enable CDX before merging? Or is there a reason to disable for now?

Reply:
oh, this was for merging into main/next release, just to be consistent in what we had before, where we disabled tmp-cdx.
It'll be reenabled in the js-warc branch.